oauth2-server/src/main/java/com/clouway/oauth2/idtoken/IdToken.java

```diff
@@ -0,0 +1,123 @@
+package com.clouway.oauth2.idtoken;
+
+import com.clouway.oauth2.jwt.Jwt;
+
+/**
+ * @author Vasil Mitov <v.mitov.clouway@gmail.com>
+ */
+public final class IdToken {
```

Comment:
ClaimSet  may have a custom claims represented as Map<String, Object>, so everyone will be able to add them if it's required.